spec/models/course/assessment_spec.rb

```diff
@@ -284,11 +284,12 @@
         create(:course_assessment_submission, assessment: assessment, creator: user1)
       end
       let(:user2) { create(:user) }
+      let(:assessment2) { create(:assessment, *assessment_traits, course: course) }
```

Comment:
A little OCD but I would think `assessment2` is for `submission2`, when it is actually for `submission3`.